composer.json

```
@@ -22,7 +22,9 @@
     ],
     "require": {
         "php": ">=8.0",
-        "ray/di": "^2.14"
+        "ray/di": "^2.14",
+        "ray/compiler": "1.x-dev",
+        "doctrine/cache": "^1.10"
```

```suggestion
        "doctrine/cache": "^1.10 || ^2.2"
```

doctrine/cache2を使ってクラアインとが他のキャッシュを選択するときに必要？